_includes/project-list.html

```diff
@@ -1,6 +1,6 @@
 <input type="text" class="myInput" id="Input" onkeyup="filterFunction()" placeholder="Search for repos..." title="Type in keyword" aria-label="repo search">
 <div class="card-deck" id="card-deck">
-  {% assign repos = site.data.repos.list | concat: site.data.temp.list | sort_natural:"name" %}
+  {% assign repos = site.data.repos.list | concat: site.data.temp.list | sort_natural:"stargazers_count" %}
```

Comment:
The stargazers are an arbitrary metric, and one that would separate related repos. I think sorting by name is sufficient. Additionally, since stargazer counts are updated statically, and this list is updated infrequently, I actually don't think we need to bother showing the stargazer count at all, but that decision was made by somebody else.